src/libsystemd-network/dhcp-option.c

```diff
@@ -14,12 +14,33 @@
 #include "strv.h"
 #include "utf8.h"
 
+/* Append type-length value structure to the options buffer */
+static int dhcp_option_append_tlv(uint8_t options[], size_t size, size_t *offset, uint8_t code, size_t optlen, const void *optval) {
+        assert(options);
+        assert(size > 0);
+        assert(offset);
+        assert(optlen < UINT8_MAX);
```

Comment:
Should `oputlen == UINT8_MAX` be ignored?? I thought it should be `<=`.

Reply:
oops